src/product/product-action.js

```diff
@@ -0,0 +1,83 @@
+import axios from "axios";
+
+export const PRODUCT_CREATE_SUCCESS = "PRODUCT_CREATE_SUCCESS";
+export const PRODUCT_FETCHED = "PRODUCT_FETCHED";
+
+function productFetched(products) {
+  return {
+    type: PRODUCT_FETCHED,
+    payload: {
+      products: products
+    }
+  };
+}
+
+//Matched endpoint with server api (events/id/tickets), to see only the tickets from that event
+export const loadProducts = () => (dispatch, getState) => {
+  axios
+    .get(`http://localhost:4000/userprofile/:id/products`)
```

Comment:
mm if you're going to use a parameter in a route then you should use it! This route reads like is going to fetch the products of one specific user, where the :id should go

Reply:
if it's a getAllProducts, then the route should just be `/products`